backend/editor/api.py

```diff
@@ -139,6 +139,16 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
     )
 
 
+@app.exception_handler(HTTPException)
+async def http_exception_handler(request: Request, exc: HTTPException):
+    """
+    Custom exception handler to log FastAPI exceptions.
+    """
+    # Log the detail message
+    log.info(f" ERROR: {exc.detail}")
+    return JSONResponse(status_code=exc.status_code, content={"message": exc.detail})
```

Comment:
https://fastapi.tiangolo.com/tutorial/handling-errors/#override-the-httpexception-error-handler

I believe we could reuse the default `http_exception_handler`

```suggestion
    return await http_exception_handler(request, exc)
```

Reply:
I'm not sure to understand what I should do, because returning  http_exception_handler creates a loop since I am overriding the HTTPException handler

Reply:
the precise link to doc is https://fastapi.tiangolo.com/tutorial/handling-errors/#re-use-fastapis-exception-handlers

So add a:
```python
import fastapi.exception_handlers
```

and here:
```python
    return await fastapi.exception_handlers.http_exception_handler(request, exc)
```

Reply:
Oh I see you resolved it indeed…